### DISCUSSION TITLE
Download display

### DISCUSSION THREAD

**Discussion by Nitwel** [directus/directus#9870] [Feature Requests]
A display that allows you to download the file from the table view. So instead of showing an image, we show an download icon with the file name that downloads the file when clicking it. We could also look into supporting file bundling in the api so you could do this on multiple files too.

**Comment by benhaynes** [comment]
My only recommendation for this type of display is to minimize the hit area for the alternative action. I want to make sure in a typical table view, most of the row is clickable for the item detail page. It should be clear that only clicking a download icon will do otherwise. I've made similar notes on the formatted value display that @alejandro-tss is working on.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
